app/graphql/sagittarius_schema.rb

```diff
@@ -5,7 +5,7 @@ class SagittariusSchema < GraphQL::Schema
   mutation(Types::MutationType)
   query(Types::QueryType)
 
-  default_max_page_size 50
+  default_max_page_size 120
```

Comment:
I think this wasn't intended to be included in the commit
```suggestion
  default_max_page_size 50
```